test/disk_size_test.rb

```diff
@@ -285,7 +305,15 @@
       disk_size = Y2Storage::DiskSize.MiB(12) / 3
       expect(disk_size.to_i).to be == 12 / 3 * 1024**2
     end
-    it "should refuse another type" do
+    it "should accept division by a float" do
```

Comment:
Well, it does make a big difference if you can increase a disk size by 20% by multiplying it by 1.2 or not. Likewise by division.

It's similar with DiskSize/DiskSize as you commented in the test - it's a big difference if the result
is expected to be an integral number of not. So IMO the tests should cover the status quo here.